pkg/ocm/clusters.go

```diff
@@ -929,6 +934,19 @@ func (c *Client) createClusterSpec(config Spec, awsClient aws.Client) (*cmv1.Clu
 		awsBuilder = awsBuilder.EtcdEncryption(cmv1.NewAwsEtcdEncryption().KMSKeyARN(config.EtcdEncryptionKMSArn))
 	}
 
+	// shared vpc
+	if config.PrivateHostedZoneID != "" {
+		awsBuilder = awsBuilder.PrivateHostedZoneID(config.PrivateHostedZoneID)
+		clusterProperties["install_config"] = "featureSet: TechPreviewNoUpgrade"
```

Comment:
We can remove this property, it applied only for the 4.14 nightly version.

Reply:
removed